src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+
+const get = async (id) => {
+  try {
```

Comment:
Naming should be representative and camelCase. I will suggest something like `getQRcode `or similar. Up to you, but remember we have many get requests.

Reply:
This is inside the qr.controller.js file and we don't have any other get here. The whole reasoning about splitting the code in different files is to not sacrifice simplicity in favor of meaningful names .  With this method we easily can follow references in the code. let's say we are in qr.controller.js so the only get would logically indicate to qr , Right?

Reply:
This is inside the qr.controller.js file and there is the only get is this one. The whole reasoning behind splitting code in different files is to not sacrifice simplicity in favor of meaningful names. Let's say we are in side the  qr.controller.js file , so the only get would logically indicate to qr , right?